docs/05_community-developer-tools/index.md

```diff
@@ -28,6 +28,8 @@ If you would like to submit a tool to be added to the list, please fill in [this
 
 * [dGoods](https://github.com/MythicalGames/dgoods): An open source and free standard for handling the virtual representation of items, both digital and physical.
 * [SimpleAssets](https://github.com/CryptoLions/SimpleAssets): A simple standard for digital assets on EOSIO blockchains: Non-Fungible Tokens (NFTs), Fungible Tokens (FTs), and Non-Transferable Tokens (NTTs).
+* [Marble](https://github.com/Dappetizer/marble-digital-items): Marble is a lightweight, highly customizable digital item standard for just about anything.
```

Comment:
for just about anything?... this sounds cheesy, cheeky; can we reformulate it to be more specific?

Reply:
I agree with Ovi, seems a little bit vague, can it make my morning tea?

Reply:
maybe "most" should be "many"